### PR TITLE
Converter: Snyk (package dependency checker (CVE))

### LINKED ISSUE #32: Converter: Snyk (package dependency checker (CVE))
Sneak (package dependency checker (CVE))

### PR DESCRIPTION
**Pull request by rx294**
Sneak (package dependency checker (CVE))
Fixes #32

### CODE REVIEW

**Comment by rx294** [comment]
Synk mapper almost complete;

Pending task:
- Parse Snyk version number from JSON

File: lib/heimdall_tools/snyk_mapper.rb [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+      rescue
+        return []
+    end
+
+    def impact(severity)
+      IMPACT_MAPPING[severity.to_sym]
+    end
+
+    def parse_mapper
+      csv_data = CSV.read(CWE_NIST_MAPPING_FILE, { encoding: 'UTF-8',
```
Comment (rbclark) [inline lib/heimdall_tools/snyk_mapper.rb:94]: This raises the following error:

```
heimdall_tools/lib/heimdall_tools/snyk_mapper.rb:94: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
.rbenv/versions/2.7.1/lib/ruby/2.7.0/csv.rb:714: warning: The called method `read' is defined here
```

This error goes away if you either remove the `{}` or if you add `**` in front of the `{}`.

File: lib/heimdall_tools/snyk_mapper.rb [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+      unique_controls
+    end
+
+
+    def to_hdf
+      project_results = {}
+      @projects.each do | project |
+        controls = []
+        project['vulnerabilities'].each do | vulnerability |
+          printf("\rProcessing: %s", $spinner.next)
```
[the remark this answers is not in the document]
Comment (rx294) [reply lib/heimdall_tools/snyk_mapper.rb:126]: fixed by commit [60f6ff7](https://github.com/mitre/heimdall_tools/pull/59/commits/60f6ff7b3e4a3b9f06461c08774032d9617b5f1f)

File: lib/heimdall_tools/snyk_mapper.rb (hunk 1 above)
Comment (rx294) [reply lib/heimdall_tools/snyk_mapper.rb]: Update [b3294fb](https://github.com/mitre/heimdall_tools/pull/59/commits/b3294fb0acce0a1e04405173f0dd19050d53b1e1)fixes the warning

**Pull request merged**